specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2018-03-01-preview/machineLearningServices.json

```diff
@@ -608,15 +608,68 @@
             }
           }
         }
+      },
```

Comment:
What's the rationale for adding a POST here? In any case, POST requires an action. This is invalid at `/workspaces/{workspaceName}/computes/{computeName}` without an action.

Reply:
It seems that the REST methods here are not consistent with REST API protocol and methods. How are these methods decided at the first place?